### ISSUE TITLE
#311 extension of copy/cloning orders

### ISSUE THREAD

**Issue by ali206** [206Tours/DB#453]
Ali -

Can you put in a separate request for this? Something along the lines of copy / clone an order.

Brad

On Mon, Jun 29, 2015 at 4:19 PM, Milanka Lachman <milanka@206tours.com> wrote:
Most of our clients are repeat customers and when they come back to book a 
new reservations I want it to be as easy as on amazon – 2 clicks and done. 
   
If they are booking a new departure / tour 
but we should not go one by one picking airports (although there should be an option)
-        Most of the data about them is same as per previous booking, 
except we will have to be diligent to check their passport / if expired 
 
I kind of 
 
 
From: Brad Atkins [mailto:brad@youreshop.com] 
Sent: Monday, June 29, 2015 4:12 PM
To: Milanka Lachman; Ali Boulanger
Subject: Re: Phase 11 #311
 
So is that like copying an existing order to a new one with some changes, or revising an existing one?
 
On Mon, Jun 29, 2015 at 4:06 PM, Milanka Lachman <milanka@206tours.com> wrote:
while you are there
-        If I have a old DB booking for a family of 5 and they are making another reservation, can you provide an option for the whole party to book same airport / insurance / additional nights 
Instead of one by one? At times, true one person will want to deviate but 99% of time it’s the whole party / family doing same. 
 
 
From: Brad Atkins [mailto:brad@youreshop.com] 
Sent: Monday, June 29, 2015 2:44 PM
To: Ali Boulanger
Cc: Milanka Lachman
Subject: Re: Phase 11 #311
 
Hey Ali -
 
This is for both the admin side and client side, allowing you (or the PAX) to add 1 or more additional PAX at once.
 
Basically it'll prompt you for the number of pax all at once, instead of having to enter the data in, hit save, wait for the DB, then enter the next info in, hit save, and so on.
 
Thanks,
Brad
